docs/training_guides/getting_set_up.rst

```diff
@@ -0,0 +1,172 @@
+===========================
+Getting Set Up With Parsons
+===========================
+
+This training guide will walk you through setting up Parsons on your computer. It provides in-depth explanations of each of the tools we recommend you use, including the command line, virtual environments, and git/version control. No prior experience should be necessary to follow this guide.
+
+You can suggest improvements to this guide or request additional guides by filing an issue in our issue tracker or telling us in Slack. To get added to our Slack, email us at *engineering@movementcooperative.org*.
+
+****************************************
+Step 1: Open Up a Command Line Interface
+****************************************
+
+Command line interfaces let you do a lot of different things on your computer, including installing and running programs and navigating the directory structure of your computer.
+
+On Macs/Linux the default command line interface is called a **Terminal** and on Windows it is called the **Command Prompt**. Command line interfaces are also sometimes called *shells*. Look for this program on your computer and open it up.
+
+The commands you can use in the command line differ somewhat dependning on whether you're using Mac/Linux or windows.
+
+**Mac/Linux**:
+
+* You can use ``pwd`` (“print working directory”) to figure out where you currently are.
+* To move around, use ``cd`` (for example ``cd ..`` means "go up one directory" or ``cd my_folder`` which means "go into ``my_folder``").
+* Use ``ls`` to list all the files and folders in your current directory.
+* A `Mac/Linux command line cheat sheet <https://www.guru99.com/linux-commands-cheat-sheet.html>`_ can help you keep track of which commands are which.
+
+**Windows**:
+
+* You can use ``cd`` to figure out where you currently are.
+* To move around, use ``cd`` (for example ``cd ..`` means "go up one directory" or ``cd my_folder`` which means "go into ``my_folder``").
+* Use ``dir`` to list all the files and folders in a directory.
+* A `Windows command line cheat sheet <http://www.cs.columbia.edu/~sedwards/classes/2017/1102-spring/Command%20Prompt%20Cheatsheet.pdf>`_ can help you keep track of which commands are which.
+
+You do not have to type everything on the command line out by hand. You can auto-complete the names of installed programs as well as files/folders in your current directory by tapping the tab key. You can also access previous commands by clicking the up and down arrows. Save your hands! Learn these tricks.
```

Comment:
Windows auto-completion is a bit more limited than Unix systems. It only autocompletes the names of files/folders in the current directory, not programs. Perhaps we should have this paragraph under both OS sections here, and then modify the Windows paragraph accordingly.

Reply:
I just re-wrote it so it was clear the tab-complete for installed programs was only for Mac/Linux. Thanks for catching that!